packages/serverless/src/domain/use-case/group.ts

```diff
@@ -1,254 +1,350 @@
+import { TGroup, TGroupDetail } from '@zkdb/common';
 import { ClientSession } from 'mongodb';
 import { getCurrentTime } from '../../helper/common.js';
 import ModelGroup from '../../model/database/group.js';
 import ModelUserGroup from '../../model/database/user-group.js';
 import ModelUser from '../../model/global/user.js';
 import { isDatabaseOwner } from './database.js';
 
-async function isGroupExist(
-  databaseName: string,
-  groupName: string,
-  session?: ClientSession
-): Promise<boolean> {
-  const modelGroup = new ModelGroup(databaseName);
+// Base TParamGroup
+type TParamGroup = TGroup & {
+  databaseName: string;
+};
 
-  const group = await modelGroup.findGroup(groupName, session);
+type TParamGroupExist = Pick<TParamGroup, 'databaseName' | 'groupName'>;
 
-  return group != null;
-}
+type TParamGroupCreate = TParamGroup;
```

Comment:
I created a new file it named `param.ts`, for common param.

Reply:
Solved, move to param from common package